NEWS

```diff
@@ -75,6 +29,7 @@
 * Added base implementation of is_reachable_v2 API using intra/inter flag
 * Introduced MD capability for non-blocking registration memory types
 #### RDMA CORE (IB, ROCE, etc.)
+* Added format is_reachable_v2 for IB interfaces
```

Comment:
Added implementation of is_reachable_v2  routine to IB interface

Reply:
Fixed.